packages/neon-core/src/wallet/multisig.ts

```diff
@@ -51,11 +51,11 @@ export function getSigningThresholdFromVerificationScript(
   verificationScript: string
 ): number {
   const checkSigOpCode = verificationScript.slice(

```

Comment:
rename var to something like InteropCode instead.

Reply:
Do you mean the `8` in `verificationScript.length - 8`?

Reply:
This isnt checking an OpCode anymore right? Its an Interop? Unless we want to combine both classes of code under the same enum.

Reply:
It is an Interop, not an opcode anymore. all interop hash length will be 8.

Reply:
so the name should be checkInteropCode, not checkSigOpCode

Reply:
OK. Changed.